bin/create-github-release.js

```diff
@@ -17,8 +17,8 @@ const TAG_VALID_REGEX = /v\d+\.\d+\.\d+/
 program.requiredOption('--tag <tag>', 'tag name to create GitHub release for')
 program.option('--repo-owner <repoOwner>', 'repository owner', 'newrelic')
 program.option(
-  '--release-notes-file <releaseNotesFile>',
-  'path to release notes file',
+  '--changelog <changelog>',
```

Comment:
i change this name to be consistent with `bin/prepare-release.js`.  I didn't realize this file had a different name when I added it to the prepare-release. Instead of going through already published workflows and update I just updated this one.